monai/_version.py

```diff
@@ -81,7 +80,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
                                  stderr=(subprocess.PIPE if hide_stderr
                                          else None))
             break
-        except EnvironmentError:
+        except OSError:
             e = sys.exc_info()[1]
```

Comment:
This is an automatically generated file, could you revert?

Reply:
reverted, and added to exluded files